pom.xml

```diff
@@ -156,6 +156,11 @@
 			<artifactId>springfox-swagger-ui</artifactId>
 			<version>${swagger.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>2.3.3.RELEASE</version>
```

Comment:
chagen to spring.boot.version property and move above to the other spring related dependencies

Reply:
Done